containers/podman
# [Fixes #11089] Switch to a new installer approach using a path manipulation helper

#### What this PR does / why we need it:

Improves the Windows installation experience and fixes a few issues:

Fixes #11089 - cleanup PATH on MSI uninstall
Additionally fixes scenarios where the path can be overwritten by setx
Also removes the console flash, since the helper is built as a silent gui
Helper executable can be rerun by user to repair PATHs broken by other tools
Utilizes executable location instead of passed parameters to remove delicate escaping requirements

#### How to verify it

1. Install podman msi
2. Check registry
```
reg query hkcu\environment /v path

HKEY_CURRENT_USER\environment
    path    REG_EXPAND_SZ   %USERPROFILE%\AppData\Local\Microsoft\WindowsApps;;C:\Program Files\RedHat\Podman
```
3. Uninstall 
4. Recheck registry

```
reg query hkcu\environment /v path

HKEY_CURRENT_USER\environment
    path    REG_EXPAND_SZ   %USERPROFILE%\AppData\Local\Microsoft\WindowsApps;

```

#### Which issue(s) this PR fixes:
Fixes #11089 

#### Special notes for your reviewer:

None
